spec/support/capybara_actions.rb

```diff
@@ -9,6 +9,11 @@ def blur_from(locator)
   def within_section(title, &)
     within(:xpath, "//section[contains(header/text(),'#{title}')]", &)
   end
+
+  # An alias so our tests are less coupled to the aXe implementation
+  def be_accessible(...)
+    be_axe_clean(...)
```

Comment:
What's the default standard used for `be_axe_clean`? I see in their docs we could specify `according_to :wcaag2a, :wcag2aa` for example.

Reply:
@lwrubel I'm just about to push up a change that makes this clearer. Thanks for flagging it!

Reply:
Looking again at the [tags in the docs](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#axe-core-tags), I think we'd want the following to be using WCAG 2.1:

```
:wcag21a
:wcag21aa
```

Reply:
I'm not sure it makes sense to test WCAG 2.0 and WCAG 2.1 standards. I think WCAG 2.1 is what our target is.

Reply:
@lwrubel Even better.